recipes/pyeee/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "pyeee" %}
+{% set version = "4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3a05b5bd639d5c9c75cb9ed4080e04a9cd88a9ef88c806f699b04e16a98c38d8
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+    - setuptools >=42
+    - setuptools_scm >=3.5
+  run:
+    - python >=3.8
+    - numpy
+    - scipy
+    - schwimmbad <=0.3.2
```

Comment:
Curious as to why the upper bound is needed

Reply:
Thanks for checking. They updated schwimmbad yesterday after 2 years and seemed to have introduced a bug (I already reported it to them). I get "UnboundLocalError: local variable 'MultiPool' referenced before assignment" when using it in the tests.

Reply:
Adrian fixed the bug. My tests run again :-) So I removed the upper bound on schwimmbad.

Reply:
Glad to hear it!

Maybe we should exclude the problematic version (unless it was also yanked from PyPI) just as a safety measure